.github/workflows/example-build.yml

```diff
@@ -0,0 +1,26 @@
+# A workflow for controlling the acions taken by CI whenever we push a commit to an open pull request
+
+name: Android SDK Example Build
+
+jobs:
+  # This workflow contains a single job for running our tests
+  test:
+    name: Run Unit Tests
+    
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
+    - uses: actions/checkout@v2
+  
+    # Sets up a JDK on the job so that we can run Java code
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    # Compile the APK for testing
+    - name: Compile APK
+      run: bash ./example :example:assembleDebug --stacktrace
+    
```

Comment:
Can you add a simple step for running unit tests here? Should be able to basically copy/paste whats in the other Github Actions workflows we made for that

Reply:
Added! At ln 28

Reply:
Just requesting one more tiny change. Where you run the test can you add `- name: Unit Tests` just before? So that each step of the pipeline is named and labelled to make it easier to understand where things fail if they do.